docs/index.md

```diff
@@ -0,0 +1,134 @@
+---
+
+# Aiven + Steampipe
+
+[Steampipe](https://steampipe.io) is an open source CLI to instantly query cloud APIs using SQL.
+
+[Aiven](https://aiven.io) is the company that offers the best data infrastructure platform to the world.
+
+For example:
+
+```sql
+select
+  id,
+  name,
+  billing_enabled,
+  tenant_id
+from
+  aiven_account;
+```
+
+```
++--------------+-------------+-----------------+-----------+
+| id           | name        | billing_enabled | tenant_id |
++--------------+-------------+-----------------+-----------+
+| a41fdc9a0625 | Turbot-test | false           | aiven     |
+| a41fd3d3b210 | turbot      | false           | aiven     |
++--------------+-------------+-----------------+-----------+
+```
+
+## Documentation
+
+- **[Table definitions & examples →](/plugins/turbot/aiven/tables)**
+
+## Get started
+
+### Install
+
+Download and install the latest Aiven plugin:
+
+```bash
+steampipe plugin install aiven
+```
+
+### Configuration
+
+Installing the latest aiven plugin will create a config file (`~/.steampipe/config/aiven.spc`) with a single connection named `aiven`:
+
+```hcl
+connection "aiven" {
+  plugin = "aiven"
+
+  # You can connect to Aiven using one of options below:
+
+  # Use API Key authentication
+  # `api_key` (required) - Create an authentication token in the Aiven Console for use with the Aiven CLI or API.
+  # To learn more about using authentication tokens, refer to https://docs.aiven.io/docs/platform/concepts/authentication-tokens
+  # Can also be set with the AIVEN_TOKEN environment variable.
+  # api_key = "oGAxUvrjAdL3QBhWnaJI67Pc9P0rPDzDfhykzVfBYPlmvVH8WdJMKaeVKzcrl4CnyXpjiaKJCCNT+OkbpxfWdDNqwZPngS"
+
+  # Use User authentication
+  # email = "test@turbot.com"
+  # password = "test@123"
+
+  # If no credentials are specified, the plugin will use Aiven CLI authentication.
+  # We recommend using API Key authentication for MFA user.
+}
+```
+
+### Authentication token Credentials
+
+You may specify the api key to authenticate:
+
+- `api_key`(required): Specify the authentication token.
+
+```hcl
+connection "aiven_via_api_key" {
+  plugin     = "aiven"
+  api_key  = "oGAxUvrjAdL3QBhWnaJI67Pc9P0rPDzDfhykzVfBYPlmvVH8WdJMKaeVKzcrl4CnyXpjiaKJCCNT+OkbpxfWdDNqwZPngS"
+}
+```
+
+### User Credentials
+
+You may specify the email ID and password to authenticate:
+
+- `email`(required): Specify the aiven email.
+- `password`(required): Specify the aiven password.
+
+```hcl
+connection "aiven_via_user" {
+  plugin     = "aiven"
+  email      = "test@turbot.com"
+  password   = "turbot@123"
+}
+```
+
+### Credentials from Environment Variables
+
+The Aiven plugin will use the Aiven environment variable to obtain credentials **only if other arguments (`api_key`, `email`, `password`) are not specified** in the connection:
+
+```sh
+export AIVEN_TOKEN="oGAxUvrjAdL3QBhWnaJI67Pc9P0rPDzDfhykzVfBYPlmvVH8WdJMKaeVKzcrl4Cny"
+```
+
+```hcl
+connection "aiven" {
+  plugin = "aiven"
+}
+```
+
+### Aiven CLI
```

Comment:
I'd move this section above the env vars section

Reply:
But we check for env vars after config creds. Should we not maintain the order?

Reply:
If we have any of the env vars set, can the CLI auth use those too?

Reply:
Yes, if you have env vars set, you can use that to login into the CLI. CLI will save that token in creds file.